lib/ancestry/materialized_path.rb

```diff
@@ -75,10 +75,11 @@ def siblings_of(object)
     end
 
     def ordered_by_ancestry(order = nil)
-      if %w(mysql mysql2 sqlite sqlite3 postgresql).include?(connection.adapter_name.downcase) && ActiveRecord::VERSION::MAJOR >= 5
+      if %w(mysql mysql2 sqlite sqlite3).include?(connection.adapter_name.downcase)
+        reorder(arel_table[ancestry_column])
+      elsif %w(sqlite sqlite3 postgresql).include?(connection.adapter_name.downcase) && ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
```suggestion
      elsif %w(postgresql).include?(connection.adapter_name.downcase) && ActiveRecord::VERSION::STRING >= "6.1"
```

This is in rails master. I'm assuming the next version is 6.1